src/value.cc

```diff
@@ -1341,7 +1341,7 @@ void value_t::in_place_not()
     break;
   }
 
-  add_error_context(_f("While applying not to %1%:") % *this);
+  add_error_context(_f("While applying 'not' to %1%:") % *this);
```

Comment:
@jwiegley The quoting within Ledger's messages is a bit inconsistent. From what I can tell most messages use single quotes and a few double quotes.
This PR maintains compatibility with previous versions.
Are you okay with deciding on one type of quotes (my preferences: 'single') and change all the double quotes to single quotes within Ledger's messages?

Reply:
Also I think there are a few messages that refer to a ledger key word (like `not` here), but do not quote it. I believe being consistent here and quoting all Ledger keywords will be helpful for users and translators.